docs/install-docker.md

```diff
@@ -241,3 +241,145 @@ make clean-conda
 make clean-bin
 ```
 
+# Mac M1 Installation
+
+### 1) Step1: Install OS Dependencies
+
+Mac users can get [docker desktop](https://docs.docker.com/desktop/mac/apple-silicon/). Also installed rosetta and used Docker Compose V2 as suggested at the moment.
+
+- **Optional**: these installations are not mentioned but might be needed:
+  - Install [brew](https://brew.sh/)
+  - Install md5sha1sum (`brew install md5sha1sum`)
+  - Install PostgreSQL (`brew install postgresql`)
+
+### Step 2: Initialize CanDIGv2 Repo
+
+```bash
+# 1. initialize repo and submodules
+git clone -b develop https://github.com/CanDIG/CanDIGv2.git
+cd CanDIGv2
+git submodule update --init --recursive
+
+# 2. copy and edit .env with your site's local configuration
+cp -i etc/env/example.env .env
+```
+
+- Edit `VENV_OS=arm64mac` for the .env file:
+
+```bash
+# options are [linux, darwin, arm64mac]
+VENV_OS=arm64mac
+VENV_NAME=candig
+```
```

Comment:
You should put the host.docker.internal thing for CANDIG_DOMAIN and CANDIG_AUTH_DOMAIN here too.

Reply:
yea I forget

Reply:
I don't see the change yet?